keps/sig-auth/3257-trust-anchor-sets/kep.yaml

```diff
@@ -8,9 +8,11 @@ participating-sigs:
 status: implementable
 creation-date: 2022-02-16
 reviewers:
-  - TBD
+  - liggitt
```

Comment:
Since you're touching it, can you please also address https://github.com/kubernetes/enhancements/pull/3258#discussion_r1030221962

Reply:
Done.